aspnet/fundamentals/localization.rst

```diff
@@ -0,0 +1,337 @@
+		await context.Response.WriteAsync($"<input type=\"button\" value=\"Create es-ES culture cookie\" onclick='createCookie();' /> ");
+		await context.Response.WriteAsync($"<input type=\"button\" value=\"Clear es-ES culture cookie\" onclick='clearCookie();' /> ");
+		await context.Response.WriteAsync(
+	@"</body>
+	</html>");
+		});
+	}
+
+As you can see from the code above, the culture is prefixed with "c=" and the UI culture is prefixed with "uic=". The two values are then delimited with a vertical bar.
+
+In order to specify a different cookie name, you simply set the ``CookieRequestCultureProvider.CookieName`` when you instantiate the ``CookieRequestCultureProvider`` object. You then specify the ``CookieRequestCultureProvider.CookieName`` when you create or clear the cookie. The following code snippet illustrates doing this.
+
+.. code-block:: c#
+	:emphasize-lines: 8,20,26
+
+	public void Configure(IApplicationBuilder app, IStringLocalizer<Startup> SR)
+	{
+		// Add the platform handler to the request pipeline.
+		app.UseIISPlatformHandler();
+
+		var options = new RequestLocalizationOptions();
+		options.RequestCultureProviders.Clear();
+		var cookieRequestCultureProvider = new CookieRequestCultureProvider { CookieName = "TestCookie" };
+		options.RequestCultureProviders.Add(cookieRequestCultureProvider);
+		app.UseRequestLocalization(options);
+
+		app.Run(async (context) =>
+		{
+			await context.Response.WriteAsync(
+	$@"<!doctype html>
+	<html>
+	<head>
+	<script>
+	function createCookie() {{
+		var cookieValue = '{cookieRequestCultureProvider.CookieName}=c=es-ES|uic=es-ES';
+		document.cookie = cookieValue;
+		window.location = window.location.href.split('?')[0];
+	}}
+
+	function clearCookie() {{
+		document.cookie='{cookieRequestCultureProvider.CookieName}=""""';
+		window.location = window.location.href.split('?')[0];
+	}}
+	</script>
+	</head>
+	<body>");
+
+		await context.Response.WriteAsync($"<h1>{SR["Hello"]}</h1>");
+		await context.Response.WriteAsync($"<input type=\"button\" value=\"Create es-ES culture cookie\" onclick='createCookie();' /> ");
+		await context.Response.WriteAsync($"<input type=\"button\" value=\"Clear es-ES culture cookie\" onclick='clearCookie();' /> ");
+		await context.Response.WriteAsync(
+	@"</body>
+	</html>");
+		});
+	}
+
+Using the AcceptLanguageHeaderRequestCultureProvider
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``AcceptLanguageHeaderRequestCultureProvider`` attempts to determine the culture by inspecting the Accept-Language value in the request header. If an Accept-Language value is not present or doesn't contain at least one culture value, ``AcceptLanguageHeaderRequestCultureProvider`` will return null. In addition, since the Accept-Language value can hold multiple cultures, the ``AcceptLanguageHeaderRequestCultureProvider.MaximumAcceptLanguageHeaderValuesToTry`` property, which defaults to 3, defines the maximum number of cultures to try.
```

Comment:
Suggest adding a link to http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.4, which defines the Accept-Language header.